### PR TITLE
Ventura

### PR DESCRIPTION
**Pull request by dhinakg**

### CODE REVIEW

File: extras/ventura.md [hunk 1]
```diff
@@ -37,13 +37,17 @@ If your SMBIOS was supported in Monterey and is not included above, you're good
 
 :::
 
+<!-- FIXTHIS
+
 For those on Haswell or Ivy Bridge, here are some simple conversions:
 
 * Ivy Bridge desktops with dGPU should use MacPro6,1
```
Comment (1Revenger1) [inline extras/ventura.md:44]: 7,1 maybe? Or iMacPro1,1

**Comment by SohelIslamImran** [comment]
Hello, @1Revenger1 and developers, I have Ryzen 5 5600G APU-based processor, MSI B450M MORTAR MAX motherboard and 16GB ram. How can I install macOS on my PC? My purpose is to install Xcode to develop IOS app. Please give me a solution. Thank you.

**Comment by spitrip82** [comment]
Hello my friends, I would love to install Ventura on my computer, but unfortunately I don't have the necessary skills to do it alone :(  I tried to go to the Olarila website  and I downloaded the Ventura iso and the EFI comet lake but at the end of the installation I always have problems , my network card is not detected and I can not connect to the internet
I tried to follow a youtuber from Brazil I tried to follow his step by step on how to configure the EFI and after installing ventura I see that it detects my network card but I can't access the internet again :( 
Could I kindly ask you to prepare me an EFI with these specifications? would make me the happiest person in the world!
I want to thank you in advance and I apologize for bothering you , a big hug from Portugal

CPU:          Intel Core i7-10700  (Processor Graphics  Intel® UHD Graphics 630) 
CHIPSET:      B460, Intel Comet Lake-S
GPU:    AMD Radeon RX 6600 // UHD Graphics 630
Network:         I225-V // 
Wireless-AC 9260

File: .vuepress/config.js [hunk 2]
```diff
@@ -280,7 +280,7 @@ module.exports = {
             children: [
                 '/extras/kaslr-fix',
                 '/extras/spoof',
-                '/extras/monterey',
+                '/extras/ventura',
```
Comment (1Revenger1) [inline .vuepress/config.js:283]: Thoughts on leaving all previous pages in addition to Ventura?

Comment (dhinakg) [reply .vuepress/config.js:283]: Eh - generally the older pages are worked into the guide by the time the new OS comes out, and the pages are still accessible through GitHub anyway, we don't delete them

**Review by 1Revenger1 — APPROVED** [review]

**Pull request merged**